assets/src/edit-story/components/movable-element/index.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * External dependencies
+ */
+import Moveable from 'react-moveable';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+import { getComponentForType } from '../../elements';
+
+const Element = styled.div`
+	cursor: pointer;
+	user-select: none;
+`;
+
+const MovableElement = ( props ) => {
+	const {
+		rotationAngle,
+		x,
+		y,
+		type,
+		selected,
+		id,
+		rest,
+	} = props;
```

Comment:
I think the new convention is to do `function` expressions. Also, destruct directly in definition as:

```js
function MovableElement( {
	rotationAngle,
	x,
	y,
	type,
	selected,
	id,
	rest,
} ) {
```

Reply:
Good to know, thanks, noted.